android/src/main/java/com/adyenreactnativesdk/configuration/ThreeDSConfigurationParser.kt

```diff
@@ -0,0 +1,31 @@
+package com.adyenreactnativesdk.configuration
+
+import com.adyen.checkout.adyen3ds2.Adyen3DS2Configuration
+import com.facebook.react.bridge.ReadableMap
+
+class ThreeDSConfigurationParser(config: ReadableMap) {
+    companion object {
+        const val TAG = "ThreeDSConfigurationParser"
+        const val THREEDS2_KEY = "threeDS2"
+        const val THREEDS2_REQUESTOR_APP_URL_KEY = "requestorAppUrl"
+    }
+
+    private var config: ReadableMap
+
+    init {
+        if (config.hasKey(THREEDS2_KEY)) {
+            this.config = config.getMap(THREEDS2_KEY)!!
+        } else {
+            this.config = config
+        }
+    }
+
+    val requestorAppUrl: String?
+        get() = if (config.hasKey(THREEDS2_REQUESTOR_APP_URL_KEY)) {
+            config.getString(THREEDS2_REQUESTOR_APP_URL_KEY)
+        } else null
```

Comment:
```suggestion
        } else {
         null
        }
```
You can decide if we should add curly braces for this case as well. The other single lines have them as well.

Reply:
I'll do full formatting in another PR :)

Reply:
Updated [bda2158](https://github.com/Adyen/adyen-react-native/pull/430/commits/bda215848c14dd61c0273291e48518f0b971e0c0)